src/Doctrine/Common/Filter/RangeFilterTrait.php

```diff
@@ -124,18 +125,18 @@ private function normalizeBetweenValues(array $values): ?array
     /**
      * Normalize the value.
      *
-     * @return int|float|null
+     * @return int|float|string | null
      */
     private function normalizeValue(string $value, string $operator)
     {
-        if (!is_numeric($value)) {
+        if (!is_numeric($value) && !UuidV6::isValid($value)) {
```

Comment:
We don't want to depend on an external vendor, like Ramsey or even Symfony UID component.
Maybe we should have our own check?